api/src/main/java/datawave/microservice/querymetric/QueryMetricsDetailListResponse.java

```diff
@@ -85,6 +86,9 @@ public String getMainContent() {
             builder.append(isJexlQuery(parameters) ? "<td style=\"white-space: pre; word-wrap: break-word;\">" : "<td style=\"word-wrap: break-word;\">")
                             .append(StringEscapeUtils.escapeHtml4(metric.getQuery())).append("</td>");
             builder.append("<td style=\"white-space: pre; word-wrap: break-word;\">").append(StringEscapeUtils.escapeHtml4(metric.getPlan())).append("</td>");
+            for (Map.Entry<String, String> entry: metric.getSubPlans().entrySet()) { // Added for subplans, holding until endpoint gets more work done
```

Comment:
I am thinking we should move these into a completely different response object.  We can keep this column in here but it should be a link to go retrieve the subplans instead of actually nesting the plans in here.